fili-core/src/test/groovy/com/yahoo/bard/webservice/data/QueryBuildingTestingResources.groovy

```diff
@@ -43,66 +45,43 @@ import com.yahoo.bard.webservice.util.SimplifiedIntervalList
 import org.joda.time.DateTimeZone
 import org.joda.time.Interval
 
-public class QueryBuildingTestingResources {
+import spock.lang.Specification
 
-    // Aggregatable dimensions
-    public Dimension d1, d2, d3, d4, d5
-    // Non-aggregatable dimensions
-    public Dimension d6, d7, d8, d9, d10
-    public LogicalMetric m1, m2, m3, m4, m5, m6
+public class QueryBuildingTestingResources extends Specification {
```

Comment:
Changed it to extend `Specification` to allow for mocking. I tried to match the original `QueryBuildingTestingResources` configuration, and checked once, but probably need an additional check. Beware if error of different testing data (should not happen but who knows).